Assignment2/juliabarrufet.ttl

```diff
@@ -0,0 +1,14 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <http://www.ontologies.org/ontologies/property#> 
+
+:Class01 :includes:Sensor029; :includes:Computer101.
+
+:Sensor029 :hasMeasurement:Measurement8401.
+
+:Measurement8401 :hasTemperature "29"^^xsd:decim; :atTime "2010-06-12T12:00:12"^^xsd:dateTime.
```

Comment:
Probably wrong with the xsd:decim? Did it parse?

Reply:
Thank you, I just updated it.